bigbluebutton-html5/imports/ui/components/user-list/chat-list-item/styles.scss

```diff
@@ -1,53 +1,50 @@
 @import "../styles.scss";
 @import "/imports/ui/stylesheets/variables/_all";
+@import "/imports/ui/stylesheets/variables/typography";
```

Comment:
Typography variables are added in app component style and CSS variables being global these series of imports aren't necessary.